collectors/plugins.d/plugins_d.c

```diff
@@ -377,6 +549,39 @@ inline size_t pluginsd_process(RRDHOST *host, struct plugind *cd, FILE *fp, int
             if(unlikely(rrdset_flag_check(st, RRDSET_FLAG_DEBUG)))
                 debug(D_PLUGINSD, "requested an END on chart %s", st->id);
 
+            if (likely(stream_stats_enabled) && (now_realtime_sec() - stream_updated > 0)) {
```

Comment:
Same here: I would probably use a an array literal of a `static struct` to define these and iterate over it, instead of repeating the `if` statements here.